### DISCUSSION TITLE
lolMiner 1.47

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1413] [General]
# lolMiner 1.47
Note: at the moment the release is Linux only, because testing the new scheme on Windows will take us some time. Be patient please :)

### Changes
- lolMiner bringing you the **fastest public Ethash LHR solver**! 77.2 - 78.2% on GDDR6 and 76 - 77% on GDDR6X Nvidia Ampere cards! Exceptions: 86% on RTX 3060 v1 with driver 460.39 or older, 55-58% on RTX 3500 and RTX 3800 12G.  Fee on the new scheme is 1%, other solvers are unchanged. Please read the important notes section below.
- Added support for LHR unlock in 470.103.01 and all 510.x drivers. In fact *we highly recommend* using these drivers, because it will give a more stable unlocker experience! 
- the parameter **--lhrtune** now takes absolute % values to fix a certain percentage of unlocking. If chosen too high, lolMiner will reduce the value to the maximum possible. The value will be tied to keep regardless of the number of locks this implies. The default is "auto" in which the miner tries to balance the number of locks and re-calibrations with the speed to achieve, hash rate might change from time to time based on unlock quality heuristics.
- Cheaper locks: when unlocking the miner will continue mining at low speed to make the unlock procedure more cheap.
-  Reduced Ton & Alephium fee in Eth+Ton / Ethash+Alephium dual mining to **0%**. You will be only paying the 1% fee on Ethash in this dual mining. This also means: no more unsafe connections to Ton servers to collect fee in case you are in a country with mining restrictions - if your own Ton connection is using Stratum over TLS. 
- New parameter **--silence** that controls the amount of information the miner will print during its work. 0 is the default behavior, 1 will turn off "new job" messages, 2 will additionally turn of the messages about found shares, 3 will leave the miner with the minute statistics only. 
- Added Nvidia memory junction temperature readings on cards that support this, e.g. GDDR6X customer GPUs, most professional and server GPUs and so on.

### Fixes
- Fixed a bug with dual mining on LHR cards where the dual algorithm was mined with reduced rate after Ethash epoch change.
- Fixed a bug with --compactaccept not showing the * sign on short statistics.

## Important notes about the new LHR unlocker scheme
- **Use 470.103.01 or 510.x drivers!** They have turned out to be much more stable in unlock in testing then older drivers.
- The miner software needs to learn certain aspects about your cards over time to optimize for the best balanced pool hash rate. The needed calibration will get better over time, but the things learned are too complex to display them or store into parameters. 
- Therefore in automatic mode the miner might start with a slightly slower hash rate at the beginning, but will get better over time. In non-automatic mode you might see more re-calibrations during the first hour of mining then normal. If the number keeps too high after that point, lower the requested % value.
- As a consequence: the longer you can keep it running the better - avoid frequent restarts of the software.
- Dual mining with Aleph & Ton is supported as well as doing ZIL dual mining. When doing ZIL make sure --enablezilcache is used on all the LHR cards, because the re-calibration is needed on every hard epoch change.
- Try to keep your settings as stable es possible. Especially: Set clocks that your GPUs can keep without running into thermal throttling. Also **use locked clock core** with no set power limit to maintain a stable GPU core clock for ideal results. Having GPUs freely changing their clocks and sometimes running into their power limit will invalidate a lot of data collected and decrease efficiency of the unlocker a lot. Same applies to memory clock - do not change it after the calibration did start. Use **--lhrwait** parameter to delay the initial calibration until you are done with your settings.
- The initial configuration takes approx 3-4 minutes from the moment the DAG was build.
- Keep away other work load from your GPU as e.g. connected screens. Also CPU mining in parallel to the new unlocker is untested and might give unexpected results.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/'>lolMiner 1.47</a>.</em>

**Comment by nirotub** [comment]
Will be the Windows version? And what drivers are recommended for it?

**Reply by r0ver2** [reply]
There will be a Windows version - but we dont have any idea about what drivers will be recommended just yet!

**Reply by nbnslider** [reply]
will test

**Comment by ovan-mining87** [comment]
just install this 1.47 version on hiveOS but stil get low Hash , actually not unlocked
even using 470.103.01 driver

**Reply by r0ver2** [reply]
Make sure you let the miner run for 5-10 minutes, to get the tuning right - then you should get proper hashrate!

**Comment by ArianHiwa** [comment]
How can install 1.47 in hive? what command can be used? thanks

**Reply by ovan-mining87** [reply]
> How can install 1.47 in hive? what command can be used? thanks

miner stop && cd /tmp && wget https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.47/lolMiner_v1.47_Lin64.tar.gz && tar -xvf lolMiner_v1.47_Lin64.tar.gz && cd 1.47 && cp lolMiner /hive/miners/lolminer/1.46a && miner start

if you want update manually .

**Reply by ArianHiwa** [reply]
thanks

**Reply by ovan-mining87** [reply]
> thanks

ok please tell me if its working fine in your rig.

**Reply by axsrza** [reply]
thanks bro, my 3 rtx3060lhr    38,5  mhs GREAT!
470.103.01 driver

**Reply by ovan-mining87** [reply]
> thanks bro, my 3 rtx3060lhr 38,5 mhs GREAT! 470.103.01 driver

nice.. can you tell me your oc setting ? bcause mine only get to 37

**Reply by 17785123713** [reply]
可以正常工作

**Comment by alex-bpc** [comment]
3060tiLHR hynix 42.5mh/s;1.6Gh/s -----> 46.8mh/s;1.4Gh/s core/mem/pl 1500/2400/190
3070ti micron 55.8mh/s;1.7Gh/s ------> 60.6mh/s;1.7Gh/s  core/mem/pl 1500/2500/250
 (hive OS)
driver 495.46
The guys are great fellows! They raised the acceleration, and my mood!

**Reply by mamafunny** [reply]
watt?

**Comment by Heivor** [comment]
When version for Windows?

**Reply by r0ver2** [reply]
Its work in progress - will come as soon as its ready!

**Comment by sweethome81** [comment]
asus tuf rtx 3070 ti ''need to re-calibrate unlocker. Doing so..''

driver 470.103.01
HiveOS
![image](https://user-images.githubusercontent.com/32683458/160229801-54e7cca5-b87f-4043-a89c-253817022f71.png)

**Reply by r0ver2** [reply]
Since its GPU0, make sure its not "active", like a monitor or HDMI dummy are plugged in. Also check your BIOS settings, I had similiar issues with a 3060Ti as GPU0 - that was solved by updating the motherboards bios and fixing the bios settings! Good luck!

**Reply by sweethome81** [reply]
I lowered the oc settings, 62 mhs works stable

**Reply by Ninety8carrots** [reply]
@r0ver2 - I am having the same issues with 3060Ti at GPU0 location. Can you give more details on how you solved the issue? Also, would simply swapping with a non-3060Ti gpu solve the problem?

**Reply by r0ver2** [reply]
> @r0ver2 - I am having the same issues with 3060Ti at GPU0 location. Can you give more details on how you solved the issue? Also, would simply swapping with a non-3060Ti gpu solve the problem?

Make sure iCPU are used a primary cpu, not sure if it makes any difference, by try set pcie gen to 2. Update bios of motherboard if there are a bios available.

**Comment by Psyxopath** [comment]
At the start, these are the errors. Why?
![изображение](https://user-images.githubusercontent.com/42670551/160247581-f9159d0a-d9db-4fea-9196-4cee23825752.png)

**Reply by r0ver2** [reply]
Defect DAG segments are normally because of too aggresive memory oc. Try reduce your memory overclock :-)

**Comment by amankhan25** [comment]
Super cool... Thank you lolminer

Previously on 3070lhr giving eth only 45 with ton but now 48

**Comment by eric8702** [comment]
Good! 3060 v1 on hive os ,how is the hashrate with 1.47 lolMiner? 
and the setting  Core, mem PL 
Thank!

**Reply by shumway66** [reply]
iam getting 42mh roughly seems about same with ton or not
![Screenshot from 2022-03-28 06 47 38](https://user-images.githubusercontent.com/40394491/160412197-1966de00-514d-41e1-9267-24f5d965975c.png)
with driver 480.39

**Reply by eric8702** [reply]
> iam getting 42mh roughly seems about same with ton or not ![Screenshot from 2022-03-28 06 47 38](https://user-images.githubusercontent.com/40394491/160412197-1966de00-514d-41e1-9267-24f5d965975c.png) with driver 480.39

in win10, my 3060 v1 has 48-49 hashrate in ETH mine. 47-48 eth hashrate and 1G ton hashrate in ETH&TON .
Run in the hive os,3060 V1 have 48 hashrate,just ETH mine?
Thank you

**Reply by r0ver2** [reply]
3060v1 can run fully unlocked in Windows if you use the 470.05 dev drivers - there are no such driver available for Linux.

**Reply by eric8702** [reply]
> 3060v1 can run fully unlocked in Windows if you use the 470.05 dev drivers - there are no such driver available for Linux.

Thanks for reply.

**Comment by nerti001** [comment]
''need to re-calibrate unlocker. Doing so..'' my hashrate before update was about 34MH and now it is at 15MH. What's the problem here?

![EA421F06-D1F0-46B4-8D37-51B59B5C5480](https://user-images.githubusercontent.com/59374670/160279924-7140d082-8b99-404c-a1db-e1ad0739eb6c.jpeg)

**Reply by eric8702** [reply]
reduce oc

**Reply by eric8702** [reply]
reduce oc

**Reply by nerti001** [reply]
Please as I'm new here, can you tell me optimal oc I have to do to fix it?

**Reply by eric8702** [reply]
3060 v1 or  v2?

**Reply by eric8702** [reply]
try to lower mem oc

**Reply by nerti001** [reply]
These are my cards I have here. 

![image](https://user-images.githubusercontent.com/59374670/160280986-f36599f3-f44c-4344-88c1-b1d01864c256.jpeg)

**Reply by nerti001** [reply]
> try to lower mem oc 

Now I have it at 2400, which is optimal rate you thinking?

**Reply by msarlak** [reply]
Set core clock to 1552 and memory clock to 2600 but leave the power limit empty

**Comment by Gamay1** [comment]
Здравствуйте RaveOs и новый lol это котастрофа, подскажите как быть? 
![16484100248591104902549249303488](https://user-images.githubusercontent.com/102485699/160297873-260489ee-b7ee-4a2f-9390-00a527f6a27d.jpg)
Хеш рейт идёт на две монеты.
![Screenshot_20220328_014059_org telegram messenger](https://user-images.githubusercontent.com/102485699/160297901-87a61556-3b72-4c98-83a8-4e3377aab4d5.jpg)
Тут 00000
Настроил правильно 
![1648410102263473153872361021843](https://user-images.githubusercontent.com/102485699/160297921-b655c6c1-527e-4b65-b14b-5b29a3ffccf2.jpg)
И вот тоже глюк 
![16484101375384563020945938683196](https://user-images.githubusercontent.com/102485699/160297952-77d971f7-ee8a-4f5d-86c6-d8b6fc18fb11.jpg)
Показывает 000 по хешрейту но он идёт

**Comment by akifhatipoglu** [comment]
Is there anyone try Rtx 3080 12GB performance?

**Reply by r0ver2** [reply]
You will get around 62mhs with a memory offset 1500 (Linux) in ETH only mode. For ETH+TON there are currently a tuning bug for 3080 12GB that will be fixed in next version - so currently the hashrate are lower than it should be for 3080 12GB in dual mode.

**Reply by akifhatipoglu** [reply]
Thank you for the detailed answer. Can an increase in the LHR rate be observed in the next versions? Something like over the '70s. I don't understand why the 12GB version of 3080 gives low mhs.

**Reply by r0ver2** [reply]
No, thats not likly. Only ETH+TON (Dual Mode) will increase. The 3080 12GB got "LHRv3", same as 3050, and dont have the same semi-unlock possibilities as the earlier versions, like 3080 10GB.

**Reply by akifhatipoglu** [reply]
Thank you very much for giving such detailed information. I'm just curious as I'm new to these things. In general, Is LHRv3 hard to crack or did nvidia make it unbreakable now? Or is it a very long process?

**Reply by r0ver2** [reply]
No idea what will be possible with LHRv3 in the future, but it looks much harder to get decent performance from it!

**Reply by ovan-mining87** [reply]
> Is there anyone try Rtx 3080 12GB performance?

LHR v3 like 3050 up to 53% performance

**Comment by Juriyx** [comment]
Hi. I have an RTX 3060 v1 and set the parameter --lhrtune 86.0 but it seems that the miner doesn't accept it and still auto-tune the GPU.
How can I set it correctly?

**Reply by r0ver2** [reply]
The miner still needs do to some tuning, but it will try to go directly to 86% unlock - so the tuning should take some less time with --lhrtune specified.

**Reply by eric8702** [reply]
> The miner still needs do to some tuning, but it will try to go directly to 86% unlock - so the tuning should take some less time with --lhrtune specified.

if 86% unlock worked,how much hashrate will be?

**Reply by Juriyx** [reply]
Thanks) I've thought so by the faster-increased value)

**Reply by r0ver2** [reply]
> 

Depends on the OC, but 3060v1 can reach 41-42 mhs, depending on OC. Remember to use the 460.39 drivers in Linux.

**Reply by Juriyx** [reply]
> if 86% unlock worked,how much hashrate will be?

41,03 MH/s but I use energy-saving core clock 1450 MHz and not the highest VRAM overclock, +1750 MHz on RaveOS

**Reply by r0ver2** [reply]
https://hashrate.no/3060#ETH got some OC and hashrate examples

**Reply by eric8702** [reply]
> > if 86% unlock worked,how much hashrate will be?
> 
> 41,03 MH/s but I use not the highest overclock, +1750 MHz

Run in win10 ,it has 48-49 hashrate，120w power 2400 memory oc

**Reply by jaazza** [reply]
> The miner still needs do to some tuning, but it will try to go directly to 86% unlock - so the tuning should take some less time with --lhrtune specified.

I can't seem to get it running consistently. Possible for you to post your flight sheet?

**Reply by ovan-mining87** [reply]
V1 must use 460.39 driver or older in hiveos

> The miner still needs do to some tuning, but it will try to go directly to 86% unlock - so the tuning should take some less time with --lhrtune specified.

**Reply by LulaSvob** [reply]
same problem here. In the instructions it says that --lhrtune range is +/-40 but then on the screen I get LHR Pct 77.2, so I have no idea what to expect and how to set this parameter. Can anyone please help if you had any success?

**Comment by mpioner** [comment]
Please implement the functionality, restoring overclocking settings to default values after the miner shuts down.

**Comment by rtpraul** [comment]
How to set OC in batch file on windows? Like on T-Rex or Gminer.

Here for lolminer I just see the option fore core clock:

--cclk arg (=*) | The core clock used for the GPUs. Cards are separated with a comma. "*" can be used to skip a card.

So how to specify memory clock, fan speed, PL and lock core clock on windows?

**Reply by SoulRipper888** [reply]
Totally agree with that. Afterburner is trash, would be great to be able to tune core / mem / pl directly in lolminer.

**Comment by fffrenia** [comment]
Not seeing 3060ti temperatures in HiveOS, not in miner screen and not in GUI after installing cryptopoo's hiveos_gddr6x_temps. Any ideas?

**Reply by r0ver2** [reply]
3060Ti do not have gddr6x memory. When thats said, its a few 3060 Tis that actually have vram temp readings, but most dont have it.

**Reply by fffrenia** [reply]
Ah, thanks.

**Reply by ovan-mining87** [reply]
> Not seeing 3060ti temperatures in HiveOS, not in miner screen and not in GUI after installing cryptopoo's hiveos_gddr6x_temps. Any ideas?

you can see on https://www.youtube.com/watch?v=aUONTFvEMUE

**Comment by mc84ss** [comment]
![lol_problem](https://user-images.githubusercontent.com/102547644/160459461-8e0e6d69-8f72-44d3-93c3-5de9e5d7e62e.jpg)

My 3060s and 3060 Ti are faster, Non-Lhr 3070 is the same, but my non-lhr 3080 is really slow and my 3070Ti is also much slower.

Both the 3080 and the 3070Ti are listed as "inf" for lhr pct.

**Reply by mc84ss** [reply]
![lol_problem_fixed](https://user-images.githubusercontent.com/102547644/160464662-6ad45438-0449-421d-95f9-91bc914c07c4.jpg)

I figured it out.  In HiveOS I added "--lhrwait 60" to my extra parameters so that my FHR cards had a chance to be detected prior to lhr autotune kicking in.

**Comment by nerti001** [comment]
HELP!

I'm using 4 RTX 3060. Please which is the optimal oc for this update because im using core 1400, mem 2600 and it is not working. My hashrate with these parameters is 10 or 15 and it's showing me everytime 'gpu need re-calibrate unlocker. doing so' sign in the monitor.

**Reply by mc84ss** [reply]
1450 core, 2300 memory is what I'm using.

**Reply by nerti001** [reply]
So what is the hashrate for one gpu in that case please?

**Reply by mc84ss** [reply]
37 Mh/s

**Reply by nerti001** [reply]
And my last question please. Do I need to update my Nvidia drivers? I'm using now 460.91.03

**Reply by mc84ss** [reply]
scroll to the top of this page and they listed the recommend drivers to use.

**Reply by ovan-mining87** [reply]
> HELP!
> 
> I'm using 4 RTX 3060. Please which is the optimal oc for this update because im using core 1400, mem 2600 and it is not working. My hashrate with these parameters is 10 or 15 and it's showing me everytime 'gpu need re-calibrate unlocker. doing so' sign in the monitor.

im use same config as you and set OC 1450 cc 2450 mc
and not set PL.

**Reply by msarlak** [reply]
Firstly you need to update your Nvidia driver version to 470.103.01 and test this settings I'm using for RTX 3060 and they will run @ ~38mhs
Core clock 1552
Memory 2600
Leave the power limit empty

**Comment by andycsm** [comment]
Ive recently installed the update on HIVEOS and noticed a huge drop, i was mining constantly for months at 162.2mh and since ive updated it fluctuates from 90mh to 150mh changing constantly, ive left it a few hours to see if it will settle but without any luck, ive also tried to downgrade back to the older version on HIVEOS and this still doesn't work or go back to how i was performing previously

any suggestions?

**Reply by rtpraul** [reply]
It's not about hiveOs version.
You need to use the previous lolminer version, probably 1.46a or 1.45. depends on what you were using.

I would first:
1. clean Flash hiveOs.
2. Update Nvidia driver. 510...
3. Load flightsheet and try again version 1.47

**Comment by psb909** [comment]
please window.ver~~~

**Comment by psb909** [comment]
please window.ver~~~!!!!!

**Reply by r0ver2** [reply]
Windows version will come when its ready! You can't rush art ;-)

**Comment by true31337** [comment]
Which is the best driver to use for HiveOS 470.103.01 or 510.x? 495.46 is currently installed.

**Reply by r0ver2** [reply]
for LHR cards, both 470.103.01 and 510.47 (or later) are similiar.

**Comment by telethrasher** [comment]
thank you, thank you, thank you....
![lolMiner 1_47 5x3060tiLHR 1xRX6700xt](https://user-images.githubusercontent.com/99104009/160706664-b3edb208-5b62-4cb0-9ae8-227e698c9d0e.png)

forgot to mention - the first card is a zotac3060tiLHR and it's having hot memory problems - need to change the pads... the next 4 are 3060tiLHR 3hynix 1samsung and the 6th card is an RX6700xt

good job on the solver !!!

**Reply by aluciano34** [reply]
Hola tengo 4 tarjetas. 3060 ti de Hynix, una 2060 y una 1660ti. Estaban funcionando correctamente, pero he actualizado la nueva versión, (0.6-213@220325 current) y mineria dual. Desde que lo he realizado, las tarjetas 3060 ti están inestables y varían su velocidad constantemente (de 11 Mhs a 37 Mhs). Cuando abro el Hive Shell , me marca este error_****### GPU 5: need to re-calibrate unlocker. Doing so...**_**
Alguien sabe que puedo hacer, he cambiado los overclooks muchas, muchas veces, pero no consigo que se estabilice.

Gracias

**Reply by JuanCruzLopezz** [reply]
Cada vez que cambies el overclock es recomendable, por lo menos en la versión 1.47, que reinicies por completo el rig para que se apliquen bien y el minero pueda calibrar la GPU. Si no llega a funcionar lo mas recomendable es que vuelvas a la versión 1.46a.

Saludos

**Comment by mth-1012** [comment]
Hey there. I'm switching a 3060v1 from Windows where I can do 50MH/s to my HiveOS rig. The miner doesn't seem to "recognize" this is a V1 card and the LHR unlocker is stuck at 77 constantly. Tried --lhrtune 86 for that specific card but always gets the warning that lhr param is too high. Running on a 1x riser btw. Any ways to make it 86% as you claim? THanks!

**Reply by Juriyx** [reply]
Have you tried 86.0 instead of 86?

**Reply by mth-1012** [reply]
> Have you tried 86.0 instead of 86?

yea. nothing changed...

**Reply by ovan-mining87** [reply]
for v1 card use 460.39 driver

**Reply by shumway66** [reply]
> for v1 card use 460.39 driver

Or older any newer wont work

**Reply by mth-1012** [reply]
hey there, thanks. i just realized i'm running 470.94. i'm willing to downgrade it to 460.39 but would it recognize (and mine with) my 3070 Ti? i checked the supported device list for 460.39, there's no 3070ti

**Reply by ovan-mining87** [reply]
> 

460.39 wont work on mixed config. 
me too use mixed config v1 with other v2 and forced to use driver other than 460.39
460.39 only dtec v1 if booting

**Comment by liaozhijie** [comment]
Have a mini BUG:

If i use GOST or SOFT ROUTER，my pool address is 127.0.0.1:xxxx and 127.0.0.1:XXXX (dual mine)，but lolminer can't connect when use dual_mine_eth_ton.bat，need open two .bat file（open eth.bat and ton.bat at the same time）
trexmienr and gminer work well，hope can fix it.
Thanks you.

**Reply by liaozhijie** [reply]
Have a mini BUG:

If i use GOST or SOFT ROUTER，my pool address is 127.0.0.1:xxxx and 127.0.0.1:XXXX (dual mine)，but lolminer can't connect when use dual_mine_eth_ton.bat，need open two .bat file（open eth.bat and ton.bat at the same time）
trexmienr and gminer work well，hope can fix it.
Thanks you.

**Comment by byramses** [comment]
For windows version though it is in beta please add a download link, I can't wait.

**Comment by JuanCruzLopezz** [comment]
Hi, I'm having problems with dual mining. One (3060 LHR) of my 8 GPUs (7 3060 LHR; 1 3060TI LHR) is not mining TON COIN at full capacity as can be seen in the attached image (GPU number 7 at 36.74Mh/s ETH - 724.98Mh/s TON ) while the other GPUs do. This problem only happens when the version of lolminer is 1.47 and always with the same GPU no matter how long it is running. The console confirms the calibration of all GPUs equally and at the same time, but then does not report any problems with that GPU.
The only solution I find so far is not to update to version 1.47, and stay on 1.46a.

Greetings.

![Sin título](https://user-images.githubusercontent.com/102705264/160954577-5e102862-9381-4b67-b74b-8d0ed0b71751.png)

**Reply by ovan-mining87** [reply]
well wait for new bug fix then

**Comment by zennan83** [comment]
Guys I'm having issue with my 3060 lhr on 1.46a window version. It keep hitting mining lock no matter which setting I tweak using afterburner. Core lock from +100 to -200, memory clock from +1000 to 1200 all the same. Anyone can help? thanks a lot.

**Reply by ovan-mining87** [reply]
dont use afterburner for set CC and MC edit in bat file

**Reply by zennan83** [reply]
Which bat file to edit? Not good in script writing... mind to teach how? now using Tonwhales dual mining.

**Reply by andrey-shostik** [reply]
> Which bat file to edit? Not good in script writing... mind to teach how? now using Tonwhales dual mining.

Nvidia inspector bat file I guess

**Reply by telethrasher** [reply]
i use nvidia-smi.exe calls in batch file to set clocking in windows... but i gave up windows because every time one of my cards crashed it took out the driver and i had to uninstall and reinstall driver so now i'm on hiveos and loving it

**Comment by CryptoMiningInsider** [comment]
If the Windows Version of lolMiner 1.47 is in BETA can you please share a Link.
I have a full LHR Rig and would be happy to help test.

**Comment by r0ver2** [comment]
There are no BETA, but there is an ALPHA version. Visit the official telegram channel, and check the pinned message: https://t.me/lolMiner

Just remember its a reason its alpha, and not a finished release :D

**Reply by CryptoMiningInsider** [reply]
Thank You!

**Comment by aluciano34** [comment]
Hi, I have 4 cards. 3060ti from Hynix, a 2060 and a 1660ti. They were working fine, but updated to the new version, (current 0.6-213@220325) and dual mined. Since he made it, the 3060 ti cards are unstable and vary their speed constantly (from 11 Mhs to 37 Mhs). When I open the Hive Shell, I get this error_****### GPU 5: You need to recalibrate the unlocker. Doing so... _
Does anyone know what I can do, I have changed the overclocks many, many times, but I can't get it to stabilize.

Thanks

**Reply by Ninety8carrots** [reply]
I have 2 3060ti with Hynix rev1.0 LHR in my set up at locations GPU0 and GPU2. I had trouble with stability of GPU0 similar to what you stated. I had to reduce my memory OC to 1600 (core lock OC currently 1500 using HiveOS) to become stabile mining ETH+TON.

I do not have LHR percentage predefined in the arguments within flight sheet.

From reading the miner documenting, with LolMiner you will need to restart the miner everytime you adjust OC settings because of the algorithms calibrating function.

In my experience, if recalibration repeatedly occurs you will need to reduce intensity of OC settings and completely restart the miner. It also may not hurt to try and restart your rig completely.

-I am using Nvidia driver version 510.*
-it may take up to 10-15 min to stabilize
-if nothing else works you can try to update the GPU bios of the unstable card with another 3060ti with Hynix that is stable

**Comment by iUniQuM** [comment]
I've been pressing f5 for a week now, but there is still no window :(

**Reply by r0ver2** [reply]
Keep on checking, you will soon find something :)

**Reply by charnet3d** [reply]
Shit I even tried GPU virtualization using WSL2 and docker on a windows host (can't really run linux native on my pcs as they're being used) but it seems mining that way is broken, whenever any miner software starts the gpu core and mem clocks just tank to 4xx Mhz which is the P8 state and stay that way until the miner is closed. Really sad.

**Reply by charnet3d** [reply]
Just noticed the comment about the alpha version above, trying it now :)

**Reply by ovan-mining87** [reply]
> l

1.48 released

**Comment by byramses** [comment]
**lol miner v1.48** Galax brand lhr v2 rtx 3060 in the latest version, my card that does not exceed 37mh band with trex now takes **39 to 40mh**.. you are great, it works very stable ... 
win 11 
nvidia driver 512.15
lol miner 1,48
msi afterburn oc settings
mclk 1560
cclk -500 or - 100 or - 150
power limit 86%
fan 70%
temp 55-56

**Reply by byramses** [reply]
![ds](https://user-images.githubusercontent.com/76632222/161341955-6e93669f-5502-404a-b715-26b820ee62ff.png)

**Reply by byramses** [reply]
![sd](https://user-images.githubusercontent.com/76632222/161342433-398cf86d-a4e0-449d-8005-6bbb95f822b4.png)

**Comment by byramses** [comment]
![124](https://user-images.githubusercontent.com/76632222/161361044-612f8990-aeca-4b8b-ae0c-e5f6e78aa62f.PNG)
124 w 38.30mh super v 1.48

**Reply by ovan-mining87** [reply]
> ![124](https://user-images.githubusercontent.com/76632222/161361044-612f8990-aeca-4b8b-ae0c-e5f6e78aa62f.PNG) 124 w 38.30mh super v 1.48

if you're only mining eth. you can underclock your cclk to -100 -200 or any minimum as posible. and only OC mmclk to 2400 or above.

**Reply by byramses** [reply]
I can't do overclock fan setting and powerlimit in windows, can you show an example? fan setting 60 , powerlimit 120watt can you write an example command

**Reply by ovan-mining87** [reply]
> I can't do overclock fan setting and powerlimit in windows, can you show an example? fan setting 60 , powerlimit 120watt can you write an example command

maybe you can set in afterburner fan seting to 60 power limit to 70 or 80 depending to your GPU max TDP. my palit has TDP 180w so I set power limit to 65 - 68 in afterburner.
im no longer using windows. because my rig only for mining.

**Reply by byramses** [reply]
--powerlimit 72 --fan 60 I'm doing something like this, I'm running the bat file as admin and I'm getting an error. How should I add a command to the afterburn standalone bat file? is there a way?

**Reply by ovan-mining87** [reply]
> --powerlimit 72 --fan 60 I'm doing something like this, I'm running the bat file as admin and I'm getting an error. How should I add a command to the afterburn standalone bat file? is there a way?

not to set command line. just apply in afterburner app itself.
ithink bat file only accept cclk & mclk command

**Reply by byramses** [reply]
> 

I understand, thank you, my friend, it often gets an error illegal cuda error in the windows environment, it happens a few times a day. We cannot repeat the settings when we are not at the computer. it would be even better if it was like other trex nbminer and similar miners

**Comment by lefshaa** [comment]
how to enable monitoring gui lolMiner API now has new page /gui, I do not understand where and what to prescribe in .bat !.yf t-rex all convenient and easy to configure, can you send a screenshot and an example of what to prescribe in it? where do I get http://<ip of your rig>:<apiport>/gui ??????

**Reply by lefshaa** [reply]
I need to monitor two regs on windows and from my phone

**Comment by vocals100** [comment]
Hi, maybe you can help me, I do not understand too much how to do this: how do you specify an ETH wallet for receiving TONCOIN mined tokens? I was told that it can be used that way because TONCOIN is an ERC20 token. Thank you.

**Comment by MizzatMining** [comment]
When it says new job received and has a value shown followed by G or when it says you found a share with a value followed by G. What does this G mean? Im new to mining pls dont roast me 😅
